index.js

```diff
@@ -19,7 +20,7 @@ var Parser = module.exports = function (format) {
         this.directives[match[1]] = i++;
         if (match[2]) {
             boundary = this.escape(match[2]);
-            regex = '([^' + boundary + ']+)' + boundary;
+            regex = '([^' + boundary + ']+)?' + boundary;
```

Comment:
The `*?` is not necessary, but it's slightly more efficient than a plain `*` because it instructs the regex engine to match characters [lazily](http://www.regular-expressions.info/repeat.html#lazy).